app.js

```diff
@@ -3,14 +3,15 @@ const express = require("express");
 const logger = require("morgan");
 const mongoose = require("mongoose");
 const cors = require("cors");
-
-const app = express();
-mongoose.connect(process.env.DB_URI);
+require("dotenv").config();
 
 const login = require("./routes/loginRoutes");
 const presentation = require("./routes/presentationRoutes");
 const slide = require("./routes/slideRoutes");
 
+const app = express();
+mongoose.connect(process.env.MONGODB_URI);
+
 const corsOptions = {
   origin: "http://localhost:5173",
```

Comment:
```suggestion
  origin: process.env.PEACHPITCH_CLIENT_URI,
```
재사용을 위해 클라이언트 URI의 환경변수화 혹은 상수화가 필요해 보입니다.